COLLABORATOR_GUIDE.md

```diff
@@ -681,13 +681,6 @@ label.
 Attach the appropriate `lts-watch-` label to any PR that may impact an LTS
 release.
 
-#### How is an LTS release cut?
```

Comment:
This also needs to be removed from the TOC.

Reply:
Thanks, done.